src/ui/shared/requests/useNotBackedUpWalletGroups.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+import type { WalletGroup } from 'src/shared/types/WalletGroup';
+import { walletPort } from 'src/ui/shared/channels';
+
+export function useNotBackedUpWalletGroups<Result = WalletGroup[]>(options?: {
+  select?: (data: WalletGroup[]) => Result;
+}) {
+  return useQuery({
+    queryKey: ['wallet/getNotBackedUpWalletGroups'],
+    queryFn: () => {
+      return walletPort.request('getNotBackedUpWalletGroups');
+    },
+    useErrorBoundary: true,
+    select: options?.select,
+  });
+}
+
+export function useNotBackedUpWalletGroupsCount() {
+  const { data: count } = useNotBackedUpWalletGroups({
+    select: (walletGroups) => walletGroups.length,
+  });
+  return count ?? 0;
```

Comment:
Can't we just return data.length?

Reply:
Or we can just remove this hook and use the other one in Settings counter. Just add .length in that hook

Reply:
✅